server/testing_service_test.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2018 Google LLC
```

Comment:
I won't stop you but I don't think these tests are actually necessary. `testingServerImpl` essentially forwards things to session.

Reply:
Agreed. Admittedly, I just wanted to keep 100% test coverage.